content/docs/tutorials/alerting_based_on_metrics.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Alerting based on metrics.
+sort_rank: 4
+---
+
+# Alerting based on metrics 
+
+In this tutorial we will create alerts on the `ping_request_count` metric that we instrumented earlier. 
+
+For the sake of this tutorial we will alert when the `ping_request_count` metric is greater than 5, Checkout real world [best practices](../practices/alerting) to learn more about alerting principles.
+
+Download the latest release of alertmanager for your operating system from [here](https://github.com/prometheus/alertmanager/releases)
+
+Alertmanager supports various recievers like `email`, `webhook`, `pagerduty`, `slack` etc through which it can notify when an alert is firing. You can find the list of recievers and how to configure them [here](../alerting/latest/configuration). We will use `webhook` as a reciever for this tutorial,head over to [webhook.site](https://webhook.site) and copy the webhook URL which we will use later to configure the alertmanager.
```

Comment:
```suggestion
Alertmanager supports various recievers like `email`, `webhook`, `pagerduty`, `slack` etc through which it can notify when an alert is firing. You can find the list of recievers and how to configure them [here](../alerting/latest/configuration). We will use `webhook` as a reciever for this tutorial, head over to [webhook.site](https://webhook.site) and copy the webhook URL which we will use later to configure the alertmanager.
```